tools/provision/formula/rocksdb.rb

```diff
@@ -3,8 +3,8 @@
 class Rocksdb < AbstractOsqueryFormula
   desc "Persistent key-value store for fast storage environments"
   homepage "http://rocksdb.org"
-  url "https://github.com/facebook/rocksdb/archive/v4.11.2.tar.gz"
-  sha256 "9374be06fdfccbbdbc60de90b72b5db7040e1bc4e12532e4c67aaec8181b45be"
+  url "https://github.com/facebook/rocksdb/archive/v5.1.4.tar.gz"
```

Comment:
@poppyseedplehzr when you have a second later this week, can you bump Window's RocksDB to 5.1.4?